recipes/sdbus-cpp/all/conanfile.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SdbusCppConan(ConanFile):
+    name = "sdbus-cpp"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Kistler-Group/sdbus-cpp"
+    description = "High-level C++ D-Bus library for Linux designed" \
+                  " to provide easy-to-use yet powerful API in modern C++"
+    topics = ("conan", "dbus", "sd-bus", "sdbus-c++", "sdbus-cpp")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_code_gen": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_code_gen": False,
+    }
+    generators = "pkg_config"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Only Linux supported")
+        if self.options.shared:
+            del self.options.fPIC
+        tools.check_min_cppstd(self, "17")
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.3")
+        if self.options.with_code_gen:
+            self.build_requires("expat/2.2.10")
+
+    def requirements(self):
+        self.requires("libsystemd/247.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        src_folder = "{}-{}".format(self.name, self.version)
+        os.rename(src_folder, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_CODE_GEN"] = self.options.with_code_gen
+        self._cmake.definitions["BUILD_DOC"] = False
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_LIBSYSTEMD"] = False
+        self._cmake.configure(source_folder=self._source_subfolder,
+                              build_folder=self._build_subfolder)
```

Comment:
It looks like you are missing the cmake wrapper which is required to inject the right flags